### PR TITLE
user manuals: add cpu burst doc

### LINKED ISSUE #91: Copy new features' manuals from Blog to Documentation
Currently, the user manuals of some features (including CPU Burst, Memory QoS, Memory Evict, ...) are archived under the blogs. It may trouble new users who want to learn or use these features.  

User manuals to add:

- [x] CPU Burst #94 
- [x] Memory QoS #100 
- TBD: Memory Evict (the feature is easy to use, maybe we'd better add documentation about implementing it via our new QoS Manager)

### PR DESCRIPTION
**Pull request by saintube**
Add the user manual for CPU Burst feature (from the Blog section).

Partially fix #91 .

Signed-off-by: saintube <saintube@foxmail.com>

### CODE REVIEW

File: docs/user-manuals/cpu-burst.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+  name: apache-demo
+  annotations:
+    koordinator.sh/cpuBurst: '{"policy": "auto"}'   # Use this annotation to enable or disable CPU Burst.
+spec:
+  containers:
+  - command:
+    - httpd
+    - -D
+    - FOREGROUND
+    image: registry.cn-zhangjiakou.aliyuncs.com/acs/apache-2-4-51-for-slo-test:v0.1
```
Comment (zwzhang0107) [inline docs/user-manuals/cpu-burst.md:151]: change to koordinator repo

File: docs/user-manuals/cpu-burst.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+- Koordinator >= 0.3
+
+### Installation
+
+Please make sure Koordinator components are correctly installed in your cluster. If not, please refer to
+[Installation](/docs/installation).
+
+### Configurations
+
+CPU Burst is enabled in koordlet by default. If not, please enable it manually by updating the feature gate in the koordlet daemonset.
```
Comment (zwzhang0107) [inline docs/user-manuals/cpu-burst.md:41]: mentioned here about feature gate, already enabled by AllBeta=true

File: docs/user-manuals/cpu-burst.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+
```
Comment (zwzhang0107) [inline docs/user-manuals/cpu-burst.md:42]: not available for BE, LSR pods

File: docs/user-manuals/cpu-burst.md (hunk 1 above)
Comment (saintube) [reply docs/user-manuals/cpu-burst.md]: done

File: docs/user-manuals/cpu-burst.md (hunk 2 above)
Comment (saintube) [reply docs/user-manuals/cpu-burst.md]: done

File: docs/user-manuals/cpu-burst.md (hunk 3 above)
Comment (saintube) [reply docs/user-manuals/cpu-burst.md:44]: done

**Pull request merged**